padglobal/padglobal.py

```diff
@@ -741,9 +776,18 @@ def which_to_text(self):
             nm = monster_id_to_named_monster(monster_id)
             if m is None or nm is None:
                 continue
-            name = nm.group_computed_basename.title()
-            grp = m.series.name
-            monsters[grp].append(name)
+            elif w.isdigit():
+                nm, _, _ = lookup_named_monster(w)
+                if nm is None:
+                    continue
+                name = nm.group_computed_basename.title()
+                m = monster_no_to_monster(nm.monster_id)
+                grp = m.series.name
+                monsters[grp].append(name)
+            else:
+                items.append(w)
```

Comment:
You incorrectly merged this change. This stuff was deliberately removed.

Reply:
I don't know what your reaction means, but this is still wrong. Where is `items` defined? That code got removed.